src/test/tools/pmemobjcli/pmemobjcli.c

```diff
@@ -391,6 +395,182 @@ parse_stage(void)
 	return stage;
 }
 
+/*
+ * pocli_reserv_id -- parse object's descriptor from action object
+ */
+static enum pocli_ret
+pocli_reserv_id(struct pocli_ctx *ctx, char *in, struct pobj_action **actp)
+{
+	char *input = strdup(in);
+	if (!input)
+		return POCLI_ERR_MALLOC;
+
+	if (!actp)
+		return POCLI_ERR_PARS;
+
+	struct pocli_args *args = pocli_args_alloc(NULL, input, ".");
+	if (!args)
+		return POCLI_ERR_PARS;
+	enum pocli_ret ret = POCLI_RET_OK;
+
+	if (strcmp(args->argv[0], "g") != 0) {
+		ret = POCLI_ERR_PARS;
+		goto out;
+	}
+	size_t size = pmemobj_root_size(ctx->pop);
+	for (int i = 1; i < args->argc; i++) {
+
+		unsigned ind;
+		char c;
+		int n = sscanf(args->argv[i], "%u%c", &ind, &c);
+		if (n != 1) {
+			ret = POCLI_ERR_PARS;
+			goto out;
+		}
+		size_t max_ind = size / sizeof(PMEMoid);
+		if (!max_ind || ind >= max_ind) {
```

Comment:
isn't max_ind in this case `MAX_ACTS`?

Reply:
fixed

Reply:
No, I mean't literally MAX_ACTS, as in `if (ind >= 1024)`.

Reply:
now should be ok